src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigure1OptionListener.java

```diff
@@ -6,11 +6,11 @@
 import edu.kis.powp.jobs2d.drivers.DriverManager;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
 
-public class SelectTestFigureOptionListener implements ActionListener {
+public class SelectTestFigure1OptionListener implements ActionListener {
```

Comment:
Unfortunately, this is not a generic solution, for each figure another class is required.